TeamsSampleHub/SampleHub/Source/wwwroot/js/home.js

```diff
@@ -1,9 +1,27 @@
-﻿function homeLogin() {
+﻿$(document).ready(function () {
+    microsoftTeams.initialize();
+    getClientSideToken()
+        .then((clientSideToken) => {
+            return getServerSideToken(clientSideToken);
+        })
+        .then((serverSideToken) => {
+            //return useServerSideToken(serverSideToken);
+        })
+        .catch((error) => {
+            if (error === "invalid_grant") {
+                // Display in-line button so user can consent
+                $("#login").show();
+            } else {
+                // Something else went wrong
```

Comment:
Consider logging the error here.  `console.log(error)`?

Reply:
Update the code to log the errors.